Tensile/BenchmarkProblems.py

```diff
@@ -150,7 +150,7 @@ def writeBenchmarkFiles(stepBaseDir, solutions, problemSizes, \
     problemType = solutions[0]["ProblemType"]
     codeObjectFiles = writeKernels( \
             globalParameters["WorkingPath"], globalParameters["CxxCompiler"], \
-            [problemType], solutions, kernels, kernelHelperOjbs, \
+            globalParameters, solutions, kernels, kernelHelperOjbs, \
```

Comment:
Are we passing in the global params now?

Reply:
The previous parameter `List[problemType]` was unused in the function. I replaced it with _params_, which, in TensileCreateLibrary main is just the _args_ dictionary. Elsewhere in the code, I replaced it with global parameters to appease the tests. This is primarily a stop-gap while we continue to migrate away from dependence on global parameters.

Reply:
Alternatively, I remove this parameter altogether and we rely on the global parameters in **writeKernels**